.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-    "python.pythonPath": "env\\Scripts\\python.exe"
+    "python.pythonPath": "C:\\Users\\sejal\\AppData\\Local\\Programs\\Python\\Python36\\python.exe"
```

Comment:
Hi @Sejal2801, Please Don't change this line in `.vscode/settings.json`
Please look into <a href="https://github.com/Feminine-Divine/Woodog/blob/master/README.md">`readme.md`</a> and there you'll get instructions on how to use a virtual environment.